wpilibcExamples/src/main/cpp/examples/RamseteCommand/cpp/RobotContainer.cpp

```diff
@@ -86,5 +86,11 @@ frc2::CommandPtr RobotContainer::GetAutonomousCommand() {
 
   return std::move(ramseteCommand)
       .BeforeStarting(
-          frc2::cmd::RunOnce([this] { m_drive.TankDriveVolts(0_V, 0_V); }, {}));
+          frc2::cmd::RunOnce(
+              [this, &exampleTrajectory] {
+                m_drive.ResetOdometry(exampleTrajectory.InitialPose());
+              },
+              {})
+              .AndThen(frc2::cmd::RunOnce(
+                  [this] { m_drive.TankDriveVolts(0_V, 0_V); }, {})));
 }
```

Comment:
This logic needs to be reordered, it was also wrong originally.
It should go:
- reset odometry
- run `ramseteCommand`
- Set drive motors to 0

Currently it runs both resetting odometry and stopping the motors in the BeforeStarting block

See the comment on java as well for the implementation